src/constants/map.js

```diff
@@ -90,7 +90,7 @@ const map = {
       annotation.Measure,
   },
   ellipseMeasurement: {
-    icon: 'ic_annotation_circle_black_24px',
+    icon: '',
```

Comment:
I think we want this to be `ic_annotation_ellipse_area_black`?

Reply:
Thanks for catching this @ZhijieZhang, I copy pasted on to the wrong place.